countwords.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+//count appreance of words in a string
+func wordCount(str string) map[string]int {
+	wordList := strings.Fields(str)
+	counts := make(map[string]int)
+	for _, word := range wordList {
+		_, ok := counts[word]
+		if ok {
+			counts[word] += 1
+		} else {
+			counts[word] = 1
+		}
+	}
+	return counts
+}
+
+func main() {
+	strLine := "ozombo unicodedeveloper node ichtrojan codebeast vscode ozombo ichtrojan twitter codebeast ozombo"
```

Comment:
we should make the variable named `sentence` and the content should be an actual sentence like `The quick brown fox jumps over the lazy dog`

Reply:
@ozombo